app/models/about.rb

```diff
@@ -98,12 +98,4 @@ def category_mods_limit
   def category_mods_limit=(number)
     @category_mods_limit = number
   end
-
-  def self.discourse_discover
-    @discourse_discover ||= {
```

Comment:
This seems to be unsafe for multisite clusters. Calling the `site/basic-info.json` endpoint seems to return results from other instances for the values stored in this memoized variable.